src/solver/conv_hip_implicit_gemm_v4.cpp

```diff
@@ -359,10 +359,21 @@ static inline ConvSolution GetSolutionBase(const ConvolutionContext& ctx,
         ctx.general_compile_options;
     // clang-format on
 
+    result.construction_params.push_back(construction_parameters);
+
     if(ctx.direction.IsForward() || ctx.direction.IsBackwardData())
         result.invoker_factory = conv::MakeImplGemmDataInvokerFactory(ctx);
+    else
+    {
+        result.invoker_factory = [](const std::vector<Kernel>& kernels) {
+            return [=](const Handle& handle, const boost::any& primitve_params) {
```

Comment:
I know im knit-picking but are you trying to say 'primitive_params'  instead of primitve_params? throughout

Reply:
Yeah, thank you. I guess I should install a spellchecker for the VS xD

Reply:
Good catch :eye: